docs/courses/grading.md

```diff
@@ -264,6 +266,10 @@ To assign comments or a grade, [Teachers Grading](/courses/classes/#teachers-gra
 
 If the students assignment has already been graded, any other teacher in the course can edit the grade with additional comments if required selecting the 'Override Grade' button.
 
+#### Extra Credit
+
+Extra credit can be given to students if required. Extra points are not added to the grade just shown as **grade + extra** in the interface for student and teacher views. The extra credit can be added manually by the teacher using the **Extra Credit** button shown above but can also be handled within an autograde script. See [Autograding Enhancements](/courses/classes/#autograding-enhancements) for more information.
```

Comment:
"Extra credit can be given to students if required. Extra credit points are not added to the grade, instead the student's score is shown as **grade + extra** in the interface. This is done because LTI does not allow a grade > 100. To make transferring extra credit points to your LMS easier, we have included a [`Has Extra Credit` filter](https://docs.codio.com/courses/grading/#the-grading-process). 

Extra credit can be added manually by the teacher using the **Extra Credit** button shown above but can also be set within an autograde script. See [Autograding Enhancements](/courses/classes/#autograding-enhancements) for more information."